<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SA_Dialog_BWSetup</class>
 <widget class="QWidget" name="SA_Dialog_BWSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*******************indoor**********************/
#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
#widget
{
	background-color:rgba(211,211,211,0.9);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
SA_LineEdit
{
	color:rgb(176,138,30);
	padding-left:10px;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #RBWAutoPB:checked,
#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #VBWAutoPB:checked
{
	color:rgba(87, 87, 87,1);
	padding: 8px 14px 7px 15px;
	background-image: url(:/sa/image/fre_checked_left.png);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #RBWAutoPB,
#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #VBWAutoPB
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
	padding: 8px 14px 7px 15px;
	background-image: url(:/sa/image/fre_checked_right.png);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #RBWManualPB:checked,
#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #VBWManualPB:checked
{
	color:rgba(87, 87, 87,1);
	padding: 8px 5px 7px 6px;
	background-image: url(:/sa/image/fre_checked_right.png);
	background-position:right;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #RBWManualPB,
#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] #VBWManualPB
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
	padding: 8px 5px 7px 6px;
	background-image: url(:/sa/image/fre_checked_left.png);
	background-position:right;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/sa/image/btn_close_normal.png);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
#pb_cancel:pressed
{
background-image: url(:/sa/image/btn_close_click.png);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
#line
{
	background-color:rgba(46,46,46,1);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
QComboBox
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color:rgba(162, 115, 0, 1);
	padding:8px 23px 7px 10px;
	combobox-popup: 0;
	border-style:flat;
	background-image: url(:/sa/image/dropdown_normal.png);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
QComboBox:focus
{
	background-image: url(:/sa/image/dropdown_click.png);
	
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
QComboBox::drop-down {
width: 0px;	border-style:flat;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
QComboBox QAbstractItemView{
   font: 75 14px &quot;Arial&quot;;
	color:rgba(0, 0, 0, 1);
	background:rgba(255,255,255,1);
 	outline:0px;
	border-style:solid;
	border-color:rgba(151, 151, 151, 1);
	border-width:1px;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
QAbstractItemView::item{
	padding-left:8px;
	width:140px;
	height:32px;
	border-style:flat;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
QComboBox QAbstractItemView::item:selected
{
 	color:rgba(240, 179, 29, 1); 
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
SA_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
SA_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}

#SA_Dialog_BWSetup[status  =  &quot;indoor&quot;] 
SA_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
/*******************outdoor**********************/
#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
#widget
{
	background-color:rgba(211,211,211,0.9);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
SA_LineEdit
{
	color:rgb(176,138,30);
	padding-left:10px;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #RBWAutoPB:checked,
#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #VBWAutoPB:checked
{
	color:rgba(87, 87, 87,1);
	padding: 8px 14px 7px 15px;
	background-image: url(:/sa/image/fre_checked_left.png);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #RBWAutoPB,
#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #VBWAutoPB
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
	padding: 8px 14px 7px 15px;
	background-image: url(:/sa/image/fre_checked_right.png);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #RBWManualPB:checked,
#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #VBWManualPB:checked
{
	color:rgba(87, 87, 87,1);
	padding: 8px 5px 7px 6px;
	background-image: url(:/sa/image/fre_checked_right.png);
	background-position:right;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #RBWManualPB,
#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] #VBWManualPB
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
	padding: 8px 5px 7px 6px;
	background-image: url(:/sa/image/fre_checked_left.png);
	background-position:right;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
background-image: url(:/sa/image/btn_close_normal.png);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
#pb_cancel:pressed
{
background-image: url(:/sa/image/btn_close_click.png);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
#line
{
	background-color:rgba(46,46,46,1);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
QComboBox
{
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	color:rgba(162, 115, 0, 1);
	padding:8px 23px 7px 10px;
	combobox-popup: 0;
	border-style:flat;
	background-image: url(:/sa/image/dropdown_normal.png);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
QComboBox:focus
{
	background-image: url(:/sa/image/dropdown_click.png);
	
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
QComboBox::drop-down {
width: 0px;	border-style:flat;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
QComboBox QAbstractItemView{
   font: 75 14px &quot;Arial&quot;;
	color:rgba(0, 0, 0, 1);
	background:rgba(255,255,255,1);
 	outline:0px;
	border-style:solid;
	border-color:rgba(151, 151, 151, 1);
	border-width:1px;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
QAbstractItemView::item{
	padding-left:8px;
	width:140px;
	height:32px;
	border-style:flat;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
QComboBox QAbstractItemView::item:selected
{
 	color:rgba(240, 179, 29, 1); 
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
SA_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
SA_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}

#SA_Dialog_BWSetup[status  =  &quot;outdoor&quot;] 
SA_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
/*******************night**********************/
#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
#widget
{
		background-color:rgba(208,182,139,0.9);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
SA_LineEdit
{
	color:rgb(176,138,30);
	padding-left:10px;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #RBWAutoPB:checked,
#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #VBWAutoPB:checked
{
	color:rgba(85,74,56,1);
	padding: 8px 14px 7px 15px;
	background-image: url(:/sa/image/night/btn_left_click.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #RBWAutoPB,
#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #VBWAutoPB
{
	color:rgba(121,106,81,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
	padding: 8px 14px 7px 15px;
	background-image: url(:/sa/image/night/btn_left_nor.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #RBWManualPB:checked,
#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #VBWManualPB:checked
{
	color:rgba(85,74,56,1);
	padding: 8px 5px 7px 6px;
	background-image: url(:/sa/image/night/btn_right_click.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #RBWManualPB,
#SA_Dialog_BWSetup[status  =  &quot;night&quot;] #VBWManualPB
{
	color:rgba(121,106,81,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
	padding: 8px 5px 7px 6px;
	background-image: url(:/sa/image/night/btn_right_nor.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
#pb_cancel
{
border-style:flat;
background-color:rgba(0,0,0,0);
border-image: url(:/sa/image/night/btn_close_normal.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
#pb_cancel:pressed
{
border-image: url(:/sa/image/night/btn_close_click.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
#line
{
	background-color:rgba(46,46,46,1);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
QComboBox
{
	background-color: rgba(251,220,167,0.60);
	color:rgba(159,98,0,1);
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
	padding:8px 23px 7px 10px;
	combobox-popup: 0;
	border-style:flat;
	background-image: url(:/sa/image/night/dropdown_normal.png);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
QComboBox:focus
{
	background-image: url(:/sa/image/night/dropdown_click.png);
	
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
QComboBox::drop-down {
width: 0px;	border-style:flat;
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
QComboBox QAbstractItemView{
   font: 75 14px &quot;Arial&quot;;
	color:rgba(0, 0, 0, 1);
	background-color: #FBDBA7;
 	outline:0px;
	border-style:solid;
	border-color:rgba(151, 151, 151, 1);
	border-width:1px;
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
QAbstractItemView::item{
	padding-left:8px;
	width:140px;
	height:32px;
	border-style:flat;
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
QComboBox QAbstractItemView::item:selected
{
 	color:rgba(159,98,0,1);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
SA_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
SA_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}

#SA_Dialog_BWSetup[status  =  &quot;night&quot;] 
SA_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>646</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Bandwidth</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="RBWManualPB">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>117</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btnRBWgroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="RBWAutoPB">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>117</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btnRBWgroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="lblRBWAuto">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>120</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RBW</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRBWValue">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>67</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RBW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VBWManualPB">
    <property name="geometry">
     <rect>
      <x>166</x>
      <y>217</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btnVBWgroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="lblVBWAuto">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>217</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>VBW</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVBWValue">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>167</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>VBW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VBWAutoPB">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>217</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btnVBWgroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="lblRatioValue">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>268</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>VBW/RBW</string>
    </property>
   </widget>
   <widget class="SA_Combobox" name="RBWBox">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>67</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxVisibleItems">
     <number>999</number>
    </property>
    <item>
     <property name="text">
      <string>5 MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Hz</string>
     </property>
    </item>
   </widget>
   <widget class="SA_Combobox" name="VBWBox">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>167</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>112</number>
    </property>
    <item>
     <property name="text">
      <string>3 MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 kHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Hz</string>
     </property>
    </item>
   </widget>
   <widget class="SA_Combobox" name="ratioBox">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>267</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxVisibleItems">
     <number>999</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.0001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.00001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.000001</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.0000001</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lblIF">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>324</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>IF OutPut</string>
    </property>
   </widget>
   <widget class="SA_Combobox" name="IF_OutPut_Box">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>323</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maxVisibleItems">
     <number>999</number>
    </property>
    <item>
     <property name="text">
      <string>OFF</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Narrow 10MHz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wide 100MHz</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SA_Combobox</class>
   <extends>QComboBox</extends>
   <header>sa_combobox.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>RBWBox</tabstop>
  <tabstop>VBWBox</tabstop>
  <tabstop>ratioBox</tabstop>
  <tabstop>RBWManualPB</tabstop>
  <tabstop>RBWAutoPB</tabstop>
  <tabstop>VBWManualPB</tabstop>
  <tabstop>VBWAutoPB</tabstop>
  <tabstop>pb_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnVBWgroup"/>
  <buttongroup name="btnRBWgroup"/>
 </buttongroups>
</ui>
